dallinger/prolific.py

```diff
@@ -350,6 +350,27 @@ def publish_study(self, study_id: str) -> dict:
             json={"action": "PUBLISH"},
         )
 
+    def screen_out(
```

Comment:
We should update this function such that it can accept either a single submission_id or a list of submission IDs.